### ISSUE TITLE
Mark items in Items tree

### ISSUE THREAD

**Issue by tomiit123** [UnderwaterApps/overlap2d#116]
Can be added: marking items in Items tree both ways when click on Image it mark image and item in Item Tree too and other way like now when mark item in Items tree correct image will be marked in workspace.

Tomas.

**Comment by azakhary** [comment]
yep should be very easy to implement, like super easy. 
Whenever you select an item, a notification is fired called ITEM_SELECTION_CHANGED. if I am not mistaken.

In the UIItemsTreeBoxMediator it is possible to add that notification in listNotificationInterests and handleNotification methods, and then when that happened and select item in tree accordingly. Should be several lines of code! :)

**Comment by tomiit123** [comment]
Yes you are right :) and I edited my first post and add second comment to it.

**Comment by azakhary** [comment]
Yeah that hovering thing would be super cool too. Although will require some modification to the tree view, to have that third state I guess. I am not sure how it works exactly.

**Comment by azakhary** [comment]
Part of this is done, item selecting highlights tree both ways. 
Should I close this one, and create separate issue for hovering?
